code/_expts/slotnick_etal_2001/analyze/plots/de_FigurizerCC.m

```diff
@@ -5,36 +5,28 @@
 % squares etc)
 
 ds = 'test';
+taskType = strcat(upper(mSets.data.taskType(1)), mSets.data.taskType(2:end));
+
 
 left = stats.rej.cc.perf.(ds){1};
 right = stats.rej.cc.perf.(ds){2};
```

Comment:
On the `right = stats.rej ...` part? Would there be a case where we have more than just 2?